pkgs/by-name/he/heliocron/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "heliocron";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "mfreeborn";
+    repo = "heliocron";
+    rev = "v${version}";
+    hash = "sha256-5rzFz29Bpy2GR6bEt2DdCq9RtpdcY3SK/KnZrBrHUvk=";
+  };
+
+  cargoHash = "sha256-RjYE2TBvHnBgiwJi/Ys+wAmJoMHJuimcD07FOqwkcis=";
+
+  meta = {
+    description = "Command line application written in Rust capable of delaying execution of other programs for time periods relative to sunrise and sunset";
```

Comment:
is it possible to have shorter description, currently `longDescription` is same length as `description`

we can borrom from arch, `Execute tasks relative to sunset, sunrise and other solar events` ?

Reply:
Sure, I just copied the description from the Github repo, I'll shorten it.